public/images/jetpack/jetpack-logo.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
There's a jetpack logo svg that is being used in the `/help` section. When that PR happened I suggested we may need a specific component soon to reuse, like `client/components/wordpress-logo`.